test/extended/testdata/bindata.go

```diff
@@ -30415,24 +30415,18 @@ os::cmd::expect_success 'oc adm policy add-cluster-role-to-group --rolebinding-n
 #os::cmd::expect_success_and_not_text 'oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth' "Warning"
 #os::cmd::expect_success 'oc adm policy add-cluster-role-to-group --rolebinding-name=self-provisioners self-provisioner system:authenticated:oauth'
 
-os::cmd::expect_success 'oc adm policy add-scc-to-user privileged fake-user'
-os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' 'fake-user'
-os::cmd::expect_success 'oc adm policy add-scc-to-user privileged -z fake-sa'
-os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' "system:serviceaccount:$(oc project -q):fake-sa"
-os::cmd::expect_success 'oc adm policy add-scc-to-group privileged fake-group'
-os::cmd::expect_success_and_text 'oc get scc/privileged -o yaml' 'fake-group'
-os::cmd::expect_success 'oc adm policy remove-scc-from-user privileged fake-user'
-os::cmd::expect_success_and_not_text 'oc get scc/privileged -o yaml' 'fake-user'
-os::cmd::expect_success 'oc adm policy remove-scc-from-user privileged -z fake-sa'
-os::cmd::expect_success_and_not_text 'oc get scc/privileged -o yaml' "system:serviceaccount:$(oc project -q):fake-sa"
-os::cmd::expect_success 'oc adm policy remove-scc-from-group privileged fake-group'
-os::cmd::expect_success_and_not_text 'oc get scc/privileged -o yaml' 'fake-group'
+os::cmd::expect_success_and_text 'oc adm policy add-scc-to-user privileged fake-user --dry-run=client -oyaml' 'fake-user'
```

Comment:
It'd be good to have at least one test that the (cluster)rolebinding was indeed created, just to make sure

Reply:
right, but that's only after https://github.com/openshift/oc/pull/412 lands, for now we need this test to accept both the old - because that's what we have for now and the new until after that oc pr lands.